### PR TITLE
Add face ID support

### PR DESCRIPTION
**Pull request by Majkitos**
- BugFix
UIActivityIndicator accessed in function stopAnimating was causing crashes because it was nil. I have added check at the top of the function.

- Face ID support
I have added assets + localized strings for face id. Main logic stays the same as for touch ID because both of these methods depend on LocalAuthentication.
I have kept names of classes the same for now to avoid possible merge conflicts but methods shared between touch and face IDs should be in the future refactored to names including biometry.

### CODE REVIEW

File: LockerUI/cs.lproj/Localizable.strings [hunk 1]
```diff
@@ -46,6 +47,7 @@
 "info-second-pin-not-match"                  = "PIN se neshoduje, opakujte zadání.";
 
 "info-input-unlock-fingerprint"              = "Použijte otisk prstu pro odemknutí.";
+"info-input-unlock-faceID"                   = "Použijte vaši tvář pro odemknutí.";
```
Comment (kirscfil) [inline LockerUI/cs.lproj/Localizable.strings]: Pro odemknutí použijte svoji tvář.

File: LockerUI/cs.lproj/Localizable.strings [hunk 2]
```diff
@@ -46,6 +47,7 @@
 "info-second-pin-not-match"                  = "PIN se neshoduje, opakujte zadání.";
 
 "info-input-unlock-fingerprint"              = "Použijte otisk prstu pro odemknutí.";
```
Comment (kirscfil) [inline LockerUI/cs.lproj/Localizable.strings]: Pro odemknutí použijte svůj otisk prstu.

File: fastlane/Fastfile [hunk 3]
```diff
@@ -63,7 +63,7 @@ platform :ios do
   end
 
   lane :pod_lint do
-    pod_lib_lint(sources: ["https://github.com/Ceskasporitelna/cocoa-pods-specs"])
+    pod_lib_lint(sources: ["https://github.com/Ceskasporitelna/cocoa-pods-specs"], allow_warnings: true)
```
Comment (kirscfil) [inline fastlane/Fastfile]: ¯\\_(ツ)_/¯

**Comment by hauseja3** [comment]
Ještě mám poznámku k public interfacu:

From docs:
```
var options = LockerUIOptions()
    options.appName = "My Awesome App"
    options.allowedLockTypes = [LockInfo(lockType: LockType.PinLock ), LockInfo(lockType: LockType.GestureLock), LockInfo(lockType: LockType.FingerprintLock), LockInfo(lockType: LockType.NoLock)]
    options.backgroundImage = UIImage(named: "myAwesomeBackground")
    options.customTint = UIColor.magentaColor()
    options.navBarColor = .default
    options.navBarTintColor = .default
    LockerUI.sharedInstance.useLockerUIOptions(options)
```

Je třeba `FingerprintLock` změnit na `BiometricLock`.

File: LockerUI/BiometricsTypeManager.swift [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+class BiometricsTypeManager {
+    
+    static let shared = BiometricsTypeManager()
+    let context = LAContext()
+    
+    func deviceUsedBiometricType() -> biometricType {
+        
+        if context.canEvaluatePolicy(LAPolicy.deviceOwnerAuthenticationWithBiometrics, error: nil) {
+            if #available(iOS 11.0, *) {
```
[the remark this answers is not in the document]
Comment (Majkitos) [reply LockerUI/BiometricsTypeManager.swift]: done

File: LockerUI/FingerprintViewController.swift [hunk 5]
```diff
@@ -88,7 +93,7 @@ class FingerprintViewController: LockerPasswordViewController
                 clog(LockerUI.ModuleName, activityName: LockerUIActivities.TouchID.rawValue, fileName: #file, functionName: #function, lineNumber: #line, logLevel: LogLevel.debug, format: "TouchID authentication succeded." )
```
[the remark this answers is not in the document]
Comment (Majkitos) [reply LockerUI/FingerprintViewController.swift]: done

File: LockerUI/cs.lproj/Localizable.strings (hunk 1 above)
Comment (Majkitos) [reply LockerUI/cs.lproj/Localizable.strings]: done

File: fastlane/Fastfile (hunk 3 above)
Comment (Majkitos) [reply fastlane/Fastfile]: I haven't changed this one intentionally, I guess Xcode did it? ¯\(ツ)/¯ Should I change it back ?

Comment (hauseja3) [reply fastlane/Fastfile]: I did

**Pull request merged**